lib/login_gov/hostdata/ec2.rb

```diff
@@ -20,16 +23,21 @@ def self.http
 
       attr_reader :document
 
+      # @param [String] document An instance identity document returned from
+      #   the EC2 metadata service at
+      #   http://169.254.169.254/2016-09-02/dynamic/instance-identity/document
       def initialize(document)
         @document = document
       end
 
+      # @return [String] Current EC2 region
       def region
-        document['region']
+        document.fetch('region')
```

Comment:
The idea here is for `#region` to raise if nil?

Reply:
To raise if the key isn't present, yeah

Reply:
:cool: